msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -17,7 +17,7 @@
     <!-- This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion -->
     <Version>$(MsalClientSemVer)</Version>
 
-    <Copyright>Copyright (c) Microsoft Corporation. All rights reserved.</Copyright>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
```

Comment:
>© [](start = 15, length = 1)

Is the recommended practice?

Reply:
![image](https://user-images.githubusercontent.com/19942418/44593555-8f7c4400-a778-11e8-94f2-2757b8fde889.png)

Reply:
the (c) is not equivalent to "©"  ?

Reply:
@jmprieur not according to the email we got. I just forwarded it to the team.